ev3dev2/motor.py

```diff
@@ -1696,6 +1758,10 @@ def wait_while(self, s, timeout=None, motors=None):
         for motor in motors:
             motor.wait_while(s, timeout)
 
+    def _block(self):
+        self.wait_until('running', timeout=WAIT_RUNNING_TIMEOUT)
```

Comment:
What's the change here? There used to be a `_block` in `MoveTank` but now that seems to have been removed and this was added. But I don't see changes to the usage.

If this is intended to replace the batch version in `MoveTank`, I have some additional questions about potential impacts.

Reply:
No change to the functionality this was just moving it up to the ``MotorSet`` class

Reply:
Can you expand on that?

It looks to me like the logic was changed from...

```
wait for motor A to start
wait for motor B to start
wait for motor A to stop
wait for motor B to stop
```

...to...

```
wait for motor A to start
wait for motor A to stop
wait for motor B to start
wait for motor B to stop
```

Which we've had issues with before.

Reply:
The logic is still
```
wait for motor A to start
wait for motor B to start
wait for motor A to stop
wait for motor B to stop
```

``MotorSet.wait_until()`` loops over all of the motors so it will wait for A to start then wait for B to start...same basic logic for ``MotorSet.wait_until_not_moving``